docs/debugging-eventlisteners.md

```diff
@@ -0,0 +1,103 @@
+<!--
+---
+linkTitle: "Debugging EventListeners"
+---
+-->
+# EventListener logs
+
+From the command-line, the easiest way to get the logs is:
+
+```shell
+$ kubectl logs deploy/el-<insert name of eventlistener> -n <namespace>
```

Comment:
There is also `tkn eventlistener logs` : https://github.com/tektoncd/cli/blob/master/docs/cmd/tkn_eventlistener_logs.md